src/main/java/uk/gov/hmcts/reform/managecase/domain/Organisation.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.managecase.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class Organisation {
+
+    private String organisationID;
```

Comment:
let me do some pedantry : )
organisationID --> organisationId

Reply:
I can't control that.. It's like that under case data json :). Otherwise we have to write another mapping config.

Reply:
we could annotate a custom json property?

Reply:
a very very minor thing - please feel free to ignore.